### ISSUE TITLE
Does runner support fixed arguments?

### ISSUE THREAD

**Issue by JRosanowski** [jessebarocio/Wox.Plugin.Runner#4]
I'd like to set up a shortcut like this

Shortcut - sdev
Path - C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\IDE\devenv.exe
Argument - E:\Dev\Something\Something.sln

I can't seem to get it to work and the only example you give is of arguments that you type into wox.

Thanks.

**Comment by jiaox99** [comment]
看了一下源码，如上配置后，只要在运行时随便多输入个字符就好了，
e.g. r sdev x

**Comment by dprice** [comment]
I am also unable to create a fixed path runner. Here's what I have:
Shortcut - `sc`
Path - `powershell.exe`
Argument - `"& 'C:\Scripts\Start-Script.ps1'"`

A PowerShell prompt is started but nothing is run. I've also tried:
Argument - `& 'C:\Scripts\Start-Script.ps1'`

You mentioned that you're using string formatting, so I'm guessing that you're doing something like this:
`string.Format("{0} {1}", "powershell.exe", "\"& 'C:\\scripts\\Start-Script.ps1'\"")`

so I even tried this:
Argument - `\"& 'C:\\Users\\derek.price\\Documents\\WindowsPowerShell\\Start-Morning.ps1'\"`

I'm able to create shortcuts to programs that don't have arguments, like Add/Remove Programs (`appwiz.cpl`), but anything with arguments just doesn't seem to work. What am I missing?

Thanks,
Derek

**Comment by eugenesvk** [comment]
I agree, a couple of examples would've been nice. Also can't get anything to work with parameters.

**Comment by liango2** [comment]
哎，都没人修复啊
